arangod/Agency/Agent.cpp

```diff
@@ -396,9 +398,11 @@ priv_rpc_ret_t Agent::recvAppendEntriesRPC(
 
 /// Leader's append entries
 void Agent::sendAppendEntriesRPC() {
+  LOG_TOPIC(INFO, Logger::AGENCY) << __FILE__<< __LINE__;
```

Comment:
This looks like debugging output which we should remove or set the log level to debug.